keras_nlp/utils/preset_utils.py

```diff
@@ -69,6 +76,14 @@ def get_file(preset, path):
             url,
             cache_subdir=os.path.join("models", subdir),
         )
+    elif preset.startswith(HF_PREFIX):
+        if huggingface_hub is None:
+            raise ImportError(
+                f"`from_preset()` requires the `huggingface_hub` package to load from '{preset}'. "
+                "Please install with `pip install huggingface_hub`."
+            )
+        hf_handle = preset.removeprefix(HF_PREFIX)
+        return huggingface_hub.hf_hub_download(repo_id=hf_handle, filename=path)
     elif os.path.exists(preset):
         # Assume a local filepath.
         return os.path.join(preset, path)
```

Comment:
Right below this line there is a error message we might want to extend.

Reply:
Thanks for the hint :+1: Updated it in https://github.com/keras-team/keras-nlp/pull/1510/commits/24ef2626032c78d5c2a02785e557e796f6599160.